### PR TITLE
Add release notes for 0.3.14

### PR DESCRIPTION
**Pull request by shannynalayna**
Signed-off-by: Shannyn Telander <telander@bitwise.io>

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,83 @@
...
+    - Make the Biome user module private.
+    - Rename `biome::datastore` to `biome::migrations` and add module-level
+      documentation for biome migrations.
+    - Remove all uses of `super` imports in Biome.
+* Correct `json-web-tokens` feature guards.
+* Experimental feature “proposal-read”:
+    - Update the `ProposalStore::proposals` arguments to take a `ProposalFilter`
+      enum to allow the proposal store to filter returned proposals.
+    - Rename the `/admin/proposals` endpoint's `filter` query parameter to `management_type`.
+    - Add `member` filter query parameter to the GET `/admin/proposals/` REST API endpoint.
```
Comment (ltseeley) [inline RELEASE_NOTES.md]: Should be "`GET /admin/proposals`" instead of "GET `/admin/proposals/`"

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,83 @@
...
+
+### CLI:
+* Experimental feature “circuit”:
+    - Update the `splinter circuit create` command to generate a service ID automatically.
+    - Add a `comments` argument to the `splinter circuit create` command to optionally
+      provide comment for the proposed circuit.
+    -  Display comments in the output of the `splinter proposals` command.
```
Comment (ltseeley) [inline RELEASE_NOTES.md]: There's an extra space here

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,83 @@
...
+    - Update the `circuit-read` and `proposal-read` response deserializers to reflect
+      the REST API response objects, rather than the related internal structs.
+    - Add a `member` argument to the `splinter circuit proposals` command for
+      filtering proposals by the given member.
+    - Add `SPLINTER_REST_API_URL` environment variable to be used if `-U` or `--url`
+      is not specified.
+    - Remove the `required` specification from `splinter circuit list` and
+      `splinter circuit proposals` arguments that are not actually required.
+* Rename the `DEFAULT_ENDPOINT` constant to `DEFAULT_SPLINTER_REST_API_URL`.
```
Comment (ltseeley) [inline RELEASE_NOTES.md]: This probably isn't necessary to mention since it doesn't change any functionality

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,83 @@
...
+* Experimental feature “circuit-template” (new): Implement the experimental
+  `circuit-template` feature. This feature lets you use a template to create a
+   circuit with the new `template` subcommand for `splinter circuit create`.
+    - Add the `dry_run` flag to the `circuit create` command, which displays the
+      circuit definition without submitting the proposal.
+    - Add subcommands behind the `circuit-template` feature to display circuit templates:
+      `splinter circuit template list` and `splinter circuit template show`.
+
+### packaging:
```
Comment (ltseeley) [inline RELEASE_NOTES.md]: Should also mention that as a part of the `circuit-template` feature, `scabbard` template is packaged with the splinter CLI and the `gameroom` template is packaged with the gameroom daemon.

Comment (shannynalayna) [reply RELEASE_NOTES.md]: Addressed.

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -1,5 +1,84 @@
...
+    - Make the Biome user module private.
+    - Rename `biome::datastore` to `biome::migrations` and add module-level
+      documentation for biome migrations.
+    - Remove all uses of `super` imports in Biome.
+* Correct `json-web-tokens` feature guards.
+* Experimental feature “proposal-read”:
+    - Update the `ProposalStore::proposals` arguments to take a `ProposalFilter`
+      enum to allow the proposal store to filter returned proposals.
+    - Rename the `/admin/proposals` endpoint's `filter` query parameter to `management_type`.
+    - Add `member` filter query parameter to the `GET /admin/proposals/` REST API endpoint.
```
Comment (ltseeley) [inline RELEASE_NOTES.md]: Looks like you still have a trailing slash for the endpoint

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -1,5 +1,84 @@
...
+   circuit with the new `template` subcommand for `splinter circuit create`.
+    - Add the `dry_run` flag to the `circuit create` command, which displays the
+      circuit definition without submitting the proposal.
+    - Add subcommands behind the `circuit-template` feature to display circuit templates:
+      `splinter circuit template list` and `splinter circuit template show`.
+
+### packaging:
+* Publish the Splinter package to crates.io when the repository is tagged.
+* Add pandoc to the `splinter-dev` image.
+* Update the version of `splinter-dev` to v2 and update Dockerfiles to use splinter-dev:v2`.
```
Comment (ltseeley) [inline RELEASE_NOTES.md]: You got a missing (or extra) "`"

File: RELEASE_NOTES.md (hunk 5 above)
Comment (shannynalayna) [reply RELEASE_NOTES.md]: Fixed

File: RELEASE_NOTES.md (hunk 6 above)
Comment (shannynalayna) [reply RELEASE_NOTES.md]: Fixed

**Pull request merged**